### PR TITLE
Handle about card error

### PR DESCRIPTION
**Pull request by punkle**
## Hey, I just made a Pull Request!

handle error in about card when refresh entity fails

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/catalog/src/components/AboutCard/AboutCard.tsx [hunk 1]
```diff
@@ -139,8 +139,15 @@ export function AboutCard(props: AboutCardProps) {
...
-    await catalogApi.refreshEntity(stringifyEntityRef(entity));
-    alertApi.post({ message: 'Refresh scheduled', severity: 'info' });
+    try {
+      await catalogApi.refreshEntity(stringifyEntityRef(entity));
+      alertApi.post({ message: 'Refresh scheduled', severity: 'info' });
+    } catch (e) {
+      alertApi.post({
+        message: `Failed to schedule refresh: ${e.message}`,
+        severity: 'error',
+      });
```
Comment (Rugvip) [inline plugins/catalog/src/components/AboutCard/AboutCard.tsx:149]: How about error API instead? :grin:

**Review by djamaile — APPROVED** [review]
🔥

File: plugins/catalog/src/components/AboutCard/AboutCard.tsx (hunk 1 above)
Comment (punkle) [reply plugins/catalog/src/components/AboutCard/AboutCard.tsx]: done.

**Review by freben — APPROVED** [review]
Nice!

**Pull request merged**